docs/css.md

```diff
@@ -50,6 +50,17 @@ app/
 @import './comments';
 ```
 
+### Importing CSS provided by an NPM package from SCSS/CSS
+
+Given your application installs an NPM package that provides CSS, such as `flatpickr`, you can import the CSS file(s) by path from the package directory within `node_modules/`:
+
+```js
+/* app/javascript/stylesheets/application.scss */
+
+@import "flatpickr/dist/flatpickr.css"
```

Comment:
This needs to have a prepending tilde, so it would be like this `@import "~flatpickr/dist/flatpickr.css"` 

Reference from issue, https://github.com/rails/webpacker/issues/2552#issuecomment-655660887.

Reply:
@nguyenj

Thanks for your comments. 

> TBH, this is actually explained already

I think this info is better explained in the CSS docs as I believe assets.md is geared more towards static assets like images; perhaps the next step is to remove CSS info from assets.md or move examples to css.md.

> This needs to have a prepending tilde

My understanding is that the prepending tilde is now incorrect since the introduction of `postcss-loader` with [`postcss-import` in the default `postcss.config.js`](https://github.com/rails/webpacker/blob/bf278f9787704ed0f78038ad7d36c008abc2edfd/lib/install/config/postcss.config.js#L3) as of [Webpacker v4.0.0.rc1](https://github.com/rails/webpacker/commit/8d496a0b871085648d6ee21c0e1e469d11cfe593). Existing documentation that refers to tilde should either be removed or notated to reflect it's relevance only to pre v4.0.0 unless I'm missing something.

I have a working demo for importing flatpickr css without the tilde prefix from the NPM package within an SCSS file running on Webpacker v5. https://github.com/rossta/rails6-webpacker-demo/compare/example/flatpickr

Reply:
@rossta 

> My understanding is that the prepending tilde is now either unnecessary or, for many projects, incorrect since the introduction of postcss-loader with postcss-import in the default postcss.config.js as of Webpacker v4.0.0.rc1. Existing documentation that refers to tilde should either be removed or notated to reflect it's relevance only to pre v4.0.0 unless I'm missing something.

This is great! Thanks for taking the time to explain. And yes, I think that assets documentation should be updated too.